### DISCUSSION TITLE
'task release' binary reports 404

### DISCUSSION THREAD

**Discussion by magichome** [semaphoreui/semaphore#1106] [General]
I've forked the semaphore repo and making some customizations.  When I build the code with 'task release' and run the associated binary for my architecture, it starts normally but every URL gives 404 "not found" error.  The official release of semaphore is 3x larger than my binary so I feel like the build is not including something.  Local builds run fine, its only the release build that gives 404.  Any suggestions would be appreciated.

**Comment by magichome** [comment]
Problem resolved.  I had messed up a config file so packr wasn't including static content into the build.

**Comment by tboerger** [comment]
You found the problem on your own so I'm closing this dicussion for now.
